phpstan.neon

```diff
@@ -0,0 +1,5 @@
+parameters:
+    checkMissingIterableValueType: false
+    paths:
+        - src/
+        - tests/
```

Comment:
To fix following error during `PHPStan` static code analyzing, using this `phpstan.neon` file to ignore `checkMissingIterableValueType` setting.

Reply:
https://github.com/phpexpertsinc/SimpleDTO/blob/master/phpstan.neon